src/main/java/com/homeaway/dropwizard/bundle/resilience4j/configuration/retry/RetryConfiguration.java

```diff
@@ -0,0 +1,98 @@
+package com.homeaway.dropwizard.bundle.resilience4j.configuration.retry;
+
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.github.resilience4j.retry.RetryConfig;
+
+public class RetryConfiguration {
+
+    /**
+     * Name for this circuit breaker for use in the registry
+     */
+    @JsonProperty
+    @NotEmpty
+    private String name;
+
+    /**
+     * Names of exceptions to ignore
+     */
+    @JsonProperty
+    private Class[] ignoreExceptions = null;
```

Comment:
Here and below are other spots where they could be `Class<? extends Throwable>[]`.